extensions/adobe/experience/customerJourneyManagement/ajo-entity-mixins.schema.json

```diff
@@ -5,131 +5,180 @@
+            "https://ns.adobe.com/experience/customerJourneyManagement/entities/channelDetails/push": {
               "title": "Push Specific Fields",
               "type": "object",
               "description": "Push Specific Fields.",
               "properties": {
-                "https://ns.adobe.com/experience/customerJourneyManagement/ajo/channel/push/title": {
+                "https://ns.adobe.com/experience/customerJourneyManagement/entities/channelDetails/push/title": {
                   "title": "Push Title",
                   "type": "string",
                   "description": "Push Title, non-personalized"
                 }
               }
             },
-            "https://ns.adobe.com/experience/customerJourneyManagement/ajo/channel/messageID": {
+            "https://ns.adobe.com/experience/customerJourneyManagement/entities/channelDetails/messageID": {
               "title": "Message ID",
               "type": "string",
-              "description": "Message ID. Does not change when message is edited."
+              "description": "Unique ID representing a non-personalized message sent to the end user."
+            },
+            "https://ns.adobe.com/experience/customerJourneyManagement/entities/channelDetails/baseMessageID": {
+              "title": "Base Message ID",
+              "type": "string",
+              "description": "Represents the base Message ID from which the published message is derived."
             },
-            "https://ns.adobe.com/experience/customerJourneyManagement/ajo/channel/messagePublicationID": {
+            "https://ns.adobe.com/experience/customerJourneyManagement/entities/channelDetails/messagePublicationID": {
               "title": "Message Publication ID",
               "type": "string",
               "description": "Message Publication ID. Represents a frozen/published version of message."
             },
-            "https://ns.adobe.com/experience/customerJourneyManagement/ajo/channel/variantID": {
+            "https://ns.adobe.com/experience/customerJourneyManagement/entities/channelDetails/variantID": {
               "title": "Channel's variant ID",
               "type": "string",
               "description": "Is not frozen and might remain same between different versions of message publication."
+            },
+            "https://ns.adobe.com/experience/customerJourneyManagement/entities/channelDetails/channel": {
+              "title": "The channel for which this row corresponds.",
+              "$ref": "https://ns.adobe.com/experience/channels/channel",
+              "type": "string",
+              "description": "Each specific experience channel defines a constant `@id`."
+            },
+            "https://ns.adobe.com/experience/customerJourneyManagement/entities/channelDetails/template": {
+              "title": "The json template used in the message",
+              "type": "string",
```

Comment:
same as above

Reply:
This one I did not understand exactly. This is not referring to a datatype.